docs/examples/webpack/webpack.config.js

```diff
@@ -17,20 +18,19 @@ module.exports = ({ WEBPACK_SERVE }, { mode }) => ({
   devServer: {
     watchFiles: {
       paths: ['**/*.html']
-    },
-    static: {
```

Comment:
By letting webpack handle assets itself they never need writing to disk during development